using_the_compiler/README.md

```diff
@@ -6,20 +6,20 @@ In the next sections a dollar sign (`$`) denotes the command line.
 
 ## Compiling and running at once
 
+Crystal files end with the `.cr` extension.
+
 To compile and run a program in a single shot you can invoke `crystal` with a single filename:
 
 ```
 $ crystal some_program.cr
 ```
 
-Alternatively you can use the `run` command:
+Alternatively, you can use the `run` command and it will do the same thing:
```

Comment:
Not sure about that. I think it's better to introduce the most minimalist form as the "default", and then clarify that actually ends up running the `run` command.